tests/Tests.FeatureManagement/Tests.FeatureManagement.csproj

```diff
@@ -18,13 +18,28 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="3.1.5" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.1.5" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.6.1" />
```

Comment:
If Microsoft.NET.Test.Sdk Version="16.6.1" is applicable only for netcoreapp3.1 and net5.0, should we move it to their respective PackageReference sections?

Reply:
Yes I think it would probably be better that way.

Reply:
Updated.